pkg/apis/deployment/v1/member_status.go

```diff
@@ -73,8 +72,7 @@ type MemberStatus struct {
 	Architecture *ArangoDeploymentArchitectureType `json:"architecture,omitempty"`
 	// Upgrade define if upgrade should be enforced during next execution
 	Upgrade bool `json:"upgrade,omitempty"`
-	// Endpoint definition how member should be reachable
-	Endpoint *string `json:"endpoint,omitempty"`
```

Comment:
Just mark as deprecated, do not remove

Reply:
It is not removed - it's moved the the deprecated section below....